### PR TITLE
Remove spam

### PR DESCRIPTION
**Pull request by Holandsoest**
- [ ] @BramvanLimpt 
- [ ] @Doge2002 
- [x] @Maucee

### CODE REVIEW

**Review by BramvanLimpt — COMMENTED** [review]
goeie

File: source/main.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
 if __name__ == "__main__":
     print('eggs')
-    print('spam')
-test
\ No newline at end of file
+    print('hoi dit is Josha')
```
Comment (BramvanLimpt) [inline source/main.py:3]: Hoi is eik nie echt goed gespedfafnald

Comment (Maucee) [reply source/main.py:3]: Ben de gij brabander ofwa

Comment (Holandsoest) [reply source/main.py:3]: Prima zo

File: source/main.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
 if __name__ == "__main__":
     print('eggs')
-    print('spam')
```
Comment (Doge2002) [inline source/main.py:3]: kut spam

**Comment by Holandsoest** [comment]
Yes very demo

**Pull request merged**